server/lib/Room.js

```diff
@@ -203,17 +219,20 @@ class Room extends EventEmitter
 		// Router media codecs.
 		const mediaCodecs = config.mediasoup.router.mediaCodecs;
 
-		const mediasoupRouters = new Map();
+		const mediasoupRouters = mafaldaRouters;
 
-		for (const worker of mediasoupWorkers)
+		for (const worker of workers)
 		{
-			const router = await worker.createRouter({ mediaCodecs });
+			// const router = await worker.createRouter({ mediaCodecs });
+			const mafaldaRouter = await mafalda.createMafaldaRouter({ mediaCodecs });
 
-			mediasoupRouters.set(router.id, router);
+			// mediasoupRouters.set(router.id, router);
+			mediasoupRouters.set(mafaldaRouter.id, mafaldaRouter);
 		}
 
+		// @TODO: How do I fetch the first router here?
```

Comment:
Please check this

Reply:
You don't need to worry about Mediasoup Routers, use a MafaldaRouter instance as if it's a big Mediasoup Router.

Reply:
I meant what is the function to get a router?